stylesheets/styles.css

```diff
@@ -1,3 +1,11 @@
 abbr[title].no-underline {
   text-decoration: none;
 }
+
+code .copy-button {
```

Comment:
I kind of like it always visible so people know it exists 🤔 but I am am willing to be told "no" lol

I do think it should be the upper right corner though, consistent with most of what I see online